index.d.ts

```diff
@@ -92,14 +104,16 @@ export interface PackumentVersion extends PackageJson {
   /**
    * packagename@versionstring
    */
-  id: string;
-  npmVersion: string;
-  nodeVersion: string;
-  npmUser: Maintainer;
+  _id: string;
+  _npmVersion: string;
+  _nodeVersion: string;
+  _npmUser: Maintainer;
   maintainers: Maintainer[];
   dist: Dist;
+  readme?: string;
+  readmeFilename?: string;
```

Comment:
`readme` and `readmeFilename` were only declared on the top-level of `Packument` before. Now they're declared on `PackageJson` and `Packument` uses `Pick<PackumentVersion, "homepage" | "bugs">`.